cloud/cluster/fetch_cron.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import (
+	"fmt"
+	"time"
+)
+
+type FetchCron struct {
```

Comment:
This file seems kind of empty. Do we really need FetchCron for this small amount of work (also I only see it used in its test file)?